static/generated/cookies.pug

```diff
@@ -24,16 +24,7 @@ html(lang="en")
       | for more details.
 
     p
-      | Compiler Explorer uses Google Analytics, which in turn uses cookies. Google Analytics allows us to see how many
-      | individual users there are, how often users visit us, the average amount of time spent on the
-      | site, the frequency and distribution of compilers selected, and so on. It is invaluable in helping us make
-      | decisions in prioritising work on the site. It's important to note that these cookies are anonymous: we don't know
-      | who each individual user is. Our Google Analytics settings prevent data being retained for more than 14 months. The
-      | cookie stored is a first party cookie: you will find it in your browser under "godbolt.org" cookies.
-
-    p
-      | Your privacy is important to us, and you may opt out of this tracking at any time using the buttons on this dialog,
-      | with no loss of access or functionality of Compiler Explorer.
+      | At the moment we are not using Cookies.
```

Comment:
```suggestion
      | Compiler Explorer does not currently use any cookies.
```

Reply:
(wording only for "we" really here, replacing the "Compiler Explorer" in the original)